packages/server-wallet/src/models/channel.ts

```diff
@@ -414,8 +418,71 @@ export class Channel extends Model implements RequiredColumns {
     return !!this.supported && this.supported.turnNum >= 2 * this.nParticipants - 1;
   }
 
-  public get isDirectFunded(): boolean {
-    return this.protocolState.directFundingStatus === 'Funded';
+  // Does the channel have funds to pay out to all allocation items?
+  public get isFullyDirectFunded(): boolean {
+    return this.isDirectFunded(true);
+  }
+
+  // Does the channel have any funds that I can claim?
+  public get isPartlyDirectFunded(): boolean {
+    return this.isDirectFunded();
+  }
+
+  private isDirectFunded(fully?: boolean): boolean {
```

Comment:
Nit: With `this.isDirectFunded(true)` it's not immediately clear what the boolean means.

If we switched it to accept a `"Fully"|"Partially"` type we'd have `this.isDirectFunded('Fully')` which I'd argue is more straightforward.

Reply:
Good point! Filed https://github.com/statechannels/statechannels/issues/3269